src/main/java/com/checkout/issuing/cards/requests/create/PhysicalCardRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.checkout.issuing.cards.requests.create;
+
+import com.checkout.issuing.cards.CardType;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class PhysicalCardRequest extends CardRequest {
+
+    @SerializedName("shipping_instructions")
+    private ShippingInstruction shippingInstructions;
+
+    @Builder
```

Comment:
In this builder, it don't include all properties such as Virtual Card, why?

Reply:
Just fixed it! Thanks